src/Controller/Api/CommitteesController.php

```diff
@@ -2,19 +2,49 @@
 
 namespace AppBundle\Controller\Api;
 
+use AppBundle\Repository\AdherentRepository;
+use AppBundle\Repository\CommitteeRepository;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
 
 class CommitteesController extends Controller
 {
     /**
      * @Route("/committees", defaults={"_enable_campaign_silence"=true}, name="api_committees")
      * @Method("GET")
      */
-    public function getApprovedCommitteesAction()
+    public function getApprovedCommitteesAction(): Response
     {
         return new JsonResponse($this->get('app.api.committee_provider')->getApprovedCommittees());
     }
+
+    /**
+     * @Route("/committees/count-for-referent-area", name="app_committees_count_for_referent_area")
+     * @Method("GET")
+     * @Security("is_granted('ROLE_REFERENT')")
+     */
+    public function committeesAndSupervisorsCountForReferentManagedAreaAction(AdherentRepository $adherentRepository, CommitteeRepository $committeeRepository): Response
+    {
+        $referent = $this->getUser();
+
+        return new JsonResponse([
+            'committees' => $committeeRepository->countApprovedCommitteesForReferentManagedArea($referent),
+            'supervisors' => $this->aggregateCount($adherentRepository->countCommitteeSupervisorsByGenderForReferentManagedArea($referent)),
+        ]);
+    }
+
+    private function aggregateCount(array $count): array
+    {
+        array_walk($count, function (&$item) {
```

Comment:
I think this loop is useless, `array_sum` will work fine without.

Reply:
It was done not for `array_sum`, but to change the representation:
```
[
    "male" => [
      "gender" => "male"
      "count" => "2"
    ],
    "female" => [
      "gender" => "female"
      "count" => "1"
    ],
]
to 
[
  "male" => 2,
  "female" => 1,
]

I will rename `aggregateCount` to `formatCount`.